recipes-app/mraa/files/0004-iot2050-Add-support-for-the-new-IOT2050-SM-variant.patch

```diff
@@ -0,0 +1,143 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Li Hua Qian <huaqian.li@siemens.com>
+Date: Wed, 19 Apr 2023 11:54:37 +0800
+Subject: [PATCH] iot2050: Add support for the new IOT2050-SM variant
+
+IOT2050-SM board was added into IOT2050 series, in contrast to existing
+variants, the IOT2050-SM board comes without a Arduino interfaces.
```

Comment:
"IOT2050-SM board was added to IOT2050 series. In contrast to existing
variants, the IOT2050-SM board comes without an Arduino interface."

Reply:
Fixed.